### PR TITLE
bump to 0.3.0

### PR DESCRIPTION
**Pull request by jasongrout**

### CODE REVIEW

**Comment by jasongrout** [comment]
@jakirkham - have you seen the circleci failure here before?

```
===== testing package: jupyterlab-0.3.0-py27_1 =====
import: 'jupyterlab'
===== jupyterlab-0.3.0-py27_1 OK =====
TEST END: jupyterlab-0.3.0-py27_1
INFO:/opt/conda/lib/python3.5/site-packages/conda_build/config.py:--keep-old-work flag not specified.  Removing source and build files.

# If you want to upload this package to anaconda.org later, type:
#
# $ anaconda upload /feedstock_root/build_artefacts/linux-64/jupyterlab-0.3.0-py27_1.tar.bz2
#
# To have conda build upload to anaconda.org automatically, use
# $ conda config --set anaconda_upload yes

Using Anaconda API: https://api.anaconda.org
Traceback (most recent call last):
  File "/feedstock_root/ci_support/upload_or_check_non_existence.py", line 124, in <module>
    main()
  File "/feedstock_root/ci_support/upload_or_check_non_existence.py", line 104, in main
    exists = built_distribution_already_exists(cli, meta, owner)
  File "/feedstock_root/ci_support/upload_or_check_non_existence.py", line 24, in built_distribution_already_exists
    fname = bldpkg_path(meta)
TypeError: bldpkg_path() missing 1 required positional argument: 'config'

./ci_support/run_docker_build.sh returned exit code 1
```

**Comment by jakirkham** [comment]
Yep, this is due to an API break in `conda-build` version `2.0.0`. We have a workaround implemented, but it will require the feedstock be updated to get it. Please try re-rendering with `conda-smithy`.

**Comment by jasongrout** [comment]
> Please try re-rendering with conda-smithy.

I've never used conda-smithy before. I'm happy to try (not right now, though), or if there is an example recipe showing the changes, I'm happy to copy over the changes by hand.

**Comment by jakirkham** [comment]
Yeah, copying is not really the way to go. Besides using `conda-smithy` is very easy. Just do the following.

1. `conda install -n root conda-smithy`
2. `cd <feedstock repo>`
3. `conda smithy rerender`
4. Add and commit all changes.
5. Push them here.

**Comment by jasongrout** [comment]
Ah, it didn't look that easy when I looked into it. Thanks! I'll try this
tomorrow.

**Comment by jasongrout** [comment]
Looks like I need to remember my anaconda account first...

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ package:
 source:
     fn: jupyterlab-{{ version }}.tar.gz
     url: https://pypi.io/packages/source/j/jupyterlab/jupyterlab-{{ version }}.tar.gz
-    md5: d7a159f8ff02d94c3aee7fe5aa4dd13d
+    md5: d6cbfed779f32ae7dcea015ccbb346c9
 
 build:
   number: 1
```
Comment (blink1073) [inline recipe/meta.yaml]: Should be 0.

**Comment by blink1073** [comment]
What happened to 7273c68?

**Comment by blink1073** [comment]
Deferring handling of entry scripts for now.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -7,7 +7,7 @@ package:
 source:
     fn: jupyterlab-{{ version }}.tar.gz
     url: https://pypi.io/packages/source/j/jupyterlab/jupyterlab-{{ version }}.tar.gz
-    md5: d7a159f8ff02d94c3aee7fe5aa4dd13d
+    md5: d6cbfed779f32ae7dcea015ccbb346c9
```
Comment (jakirkham) [inline recipe/meta.yaml]: Also the new [PyPI site]( https://pypi.org/ ) has `sha256` checksums. So that might be useful for stronger validation.

**Comment by jakirkham** [comment]
The re-rendering looks good. Did that go reasonably well, @jasongrout?

**Pull request merged**